woocommerce/woocommerce-gutenberg-products-block
# Improve Hero Product Chessboard pattern

## What

Fixes #11404. This PR improves:
- spacing
- text size

## Why



## Testing Instructions



_Please consider any edge cases this change may have, and also other areas of the product this may impact._

1. Add a new post.
2. Add the `Hero Product Chessbord` pattern.
3. Save the page.
4. Visit the saved page
5. Ensure that the pattern looks like in the images below.


## Screenshots or screencast



### Mockup

| Desktop |Mobile |
| ------ | ----- |
|![image](https://github.com/woocommerce/woocommerce-blocks/assets/4463174/a98d0696-78c8-4845-8f5d-ef8ce7ddca45)|![image](https://github.com/woocommerce/woocommerce-blocks/assets/4463174/1df4926e-d452-4502-8b9b-d2ccf0fff0a3)|


### Implementation

| Desktop | Mobile |
|--------|--------|
|![locale local](https://github.com/woocommerce/woocommerce-blocks/assets/4463174/ab742211-4b4a-479d-87f9-75c0d1329e32)|!['Locale Galaxy Fold](https://github.com/woocommerce/woocommerce-blocks/assets/4463174/832ea72f-21ff-4070-bee6-341e3732fe65)| 

## WooCommerce Visibility


Required:


## Checklist

Required:

Conditional:

## Changelog


> Add suggested changelog entry here.
